packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -40,7 +40,7 @@ class RenderProxyBox extends RenderBox with RenderObjectWithChildMixin<RenderBox
   /// Proxy render boxes are rarely created directly because they simply proxy
   /// the render box protocol to [child]. Instead, consider using one of the
   /// subclasses.
-  RenderProxyBox([RenderBox child = null]) {
+  RenderProxyBox([RenderBox child = null]) { //ignore: avoid_init_to_null
```

Comment:
Ok, I have a repro.  Splitting my minimal example into separate files reflecting the file layout of the actual code breaks the analysis server.  Issue filed here:

https://github.com/dart-lang/sdk/issues/30328